src/pages/MovieDetails.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
I definitely think that `MovieDetails`page and `Home` should be merged into one page. Thus you will avoid code duplication (`<TopContainer />`, `<FilterPanel />`, `<MoviesList />`)

Reply:
fixed